doc/classes/MultiMesh.xml

```diff
@@ -54,7 +54,7 @@
 			<argument index="1" name="color" type="Color" />
 			<description>
 				Sets the color of a specific instance by [i]multiplying[/i] the mesh's existing vertex colors.
-				For the color to take effect, ensure that [member use_colors] is [code]true[/code] on the [MultiMesh] and [member BaseMaterial3D.vertex_color_use_as_albedo] is [code]true[/code] on the material.
+				For the color to take effect, ensure that [member use_colors] is [code]true[/code] on the [MultiMesh], [member BaseMaterial3D.vertex_color_use_as_albedo] is [code]true[/code] on the material and albedo color of the material is set to pure white R: 255, G: 255, B: 255
```

Comment:
I would reword this as it's not necessarily incorrect to have a non-white albedo color:

```suggestion
				For the color to take effect, ensure that [member use_colors] is [code]true[/code] on the [MultiMesh], [member BaseMaterial3D.vertex_color_use_as_albedo] is [code]true[/code] on the material. If the color doesn't look as expected, make sure the material's albedo color is set to pure white ([code]Color(1, 1, 1)[/code]).
```